source/RialtoGStreamerWebAudioSink.cpp

```diff
@@ -142,6 +144,8 @@ static gboolean rialto_web_audio_sink_send_event(GstElement *element, GstEvent *
 static GstStateChangeReturn rialto_web_audio_sink_change_state(GstElement *element, GstStateChange transition)
 {
     RialtoWebAudioSink *sink = RIALTO_WEB_AUDIO_SINK(element);
+    RialtoWebAudioSinkPrivate *priv = sink->priv;
+    const std::shared_ptr<GStreamerWebAudioPlayerClient> &kClient = priv->m_webAudioClient;
```

Comment:
Probably not so important for webaudio, as the lifecycle of the webaudioclient covers init-?finalize